processing/src/main/java/io/druid/segment/Rowboat.java

```diff
@@ -126,6 +126,11 @@ public int compareTo(Rowboat rhs)
     return retVal;
   }
 
+  public int compareTimestamp(Rowboat rhs)
```

Comment:
can you just have Longs.compare(left.timestamp, right.timestamp) and avoid creating this function?